test/integ_tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+amazon-braket-sdk
```

Comment:
I don't think we need to install stuff like tensorflow or torch for these tests. I'd like to use this for my github actions as well, so it would be good to keep a minimum list.

We can test out how it builds with the yml file as part of the build pipeline (when we actually test it in the NBI).